pkg/networkutils/network.go

```diff
@@ -329,6 +359,24 @@ func (n *linuxNetwork) SetupHostNetwork(vpcCIDR *net.IPNet, vpcCIDRs []*string,
 		rule:        snatRule,
 	})
 
+	var snatStaleRulesToClear []iptablesRule
+	log.Debugf("Setup Host Network: synchronising SNAT stale rules")
+	for _, staleRule := range snatStaleRulesToCheck {
+		keepRule := false
+		for _, newRule := range iptableRules {
+			if staleRule.chain == newRule.chain && reflect.DeepEqual(newRule.rule, staleRule.rule) {
```

Comment:
`reflect.DeepEqual()` is a very slow way to compare two string slices, but I agree that it's fine to use here since we only set up the host networking once.

Reply:
I am glad you agree. I wasn't too sure about keeping it like this, but writing a custom comparison just seem a bit over the top for this use case.